### DISCUSSION TITLE
how to build the examples

### DISCUSSION THREAD

**Discussion by esigo** [bazelbuild/rules_foreign_cc#833] [Q&A]
Hi,
Thanks for the great work.
I was wondering how the examples can be consumed.
I'm interested in building examples/third_party/openssl on Windows.
`bazel build //examples/third_party/openssl/...` doesn't seem to work.
Thanks

**Comment by jsharpe** [comment]
Sorry for the delay in replying.
The examples are their own WORKSPACE so you need to cd into the `examples` directory and then run `bazel build` from there. 
The examples in `third_party` are then loaded into their own repositories so to build openssl you will need to run `bazel build @openssl//...`
